ndmg/scripts/ndmg_bids.py

```diff
@@ -334,23 +316,13 @@ def main():
     parser.add_argument(
         "output_dir",
         help="The directory where the output "
-        "files should be stored. If you are running group "
-        "level analysis this folder should be prepopulated "
-        "with the results of the participant level analysis.",
-    )
-    parser.add_argument(
-        "analysis_level",
-        help="Level of the analysis that "
-        "will be performed. Multiple participant level "
-        "analyses can be run independently (in parallel) "
-        "using the same output_dir.",
-        choices=["participant", "group"],
-        default="participant",
+        "files should be stored.",
     )
     parser.add_argument(
         "--modality",
-        help="Modality of MRI scans that \
-                        are being evaluated.",
+        help="Modality of MRI scans that"
+            " are being evaluated. Functional modality"
+            " analysis still under development.",
```

Comment:
Great job! Looks like it passes on travis. One minor change (that might be slightly nitpicky): 
if somebody tries to pass `--modality func`, could you have the pipeline raise `NotImplementedError`?
https://docs.python.org/3/library/exceptions.html#NotImplementedError

Other than that, I think we're good -- thanks!

Reply:
Good point, added it in and am waiting on travis